components/director/pkg/graphql/schema.graphql

```diff
@@ -253,9 +253,14 @@ input ApplicationRegisterInput {
 	**Validation:** valid URL, max=256
 	"""
 	healthCheckURL: String
-	bundles: [BundleCreateInput!]
+	"""
+	**Validation:** valid URL, max=256
+	"""
+	baseUrl: String
 	integrationSystemID: ID
+	systemNumber: String
```

Comment:
Why do we need to allow setting systemNumber from outside? Can we trust the client for such information?